dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1199,6 +1199,35 @@ private void publishRelatedLinks(final Contentlet contentlet,
         }
     } // publishRelatedLinks.
 
+    @Override
+    public PaginatedContentList<Contentlet> searchPaginatedByPage(final String luceneQuery,
+            final int contentsPerPage, int page, final String sortBy, final User user,
+            final boolean respectFrontendRoles) throws DotDataException, DotSecurityException {
+
+        // Calculate the offset
+        if (page < 1) {
```

Comment:
I think it would be great to be such as MathUtils.ifPositive(page, 1)
Also the input arguments should be read only